draft-ietf-tls-tls13.md

```diff
@@ -3067,9 +3046,9 @@ for each scenario:
 
 | Mode | Handshake Context | Base Key |
 |------|-------------------|----------|
-| 0-RTT | ClientHello | xSS |
-| 1-RTT (Server) | ClientHello ... later of EncryptedExtensions/CertificateRequest | master secret |
-| 1-RTT (Client) | ClientHello ... ServerFinished     | master secret |
+| 0-RTT | ClientHello | Early Secret |
```

Comment:
nit: capitalization inconsistency; either use lowercase here or uppercase for each line below